docs/developer_guide/add_a_combiner.md

```diff
@@ -43,16 +43,16 @@ template. At a high level, to add a new combiner:
 
 ## 1. Define combiner configuration
 
-The combiner configuration is a `dataclass` whose properties are the configuration parameters of the combiner. All
-fields should have a type and a default value. The `ludwig.utils.schema_utils.py` module provides convenience methods
-for specifying the valid types and ranges of a combiner config. For example, the `TransformerCombiner` has the following
-config schema:
+The combiner configuration is a `dataclass` (that must extend `BaseCombinerConfig`) whose properties are the configuration
+parameters of the combiner. All fields should have a type and a default value. The `ludwig.utils.schema_utils.py` module
+provides convenience methods for specifying the valid types and ranges of a combiner config. For example, the
+`TransformerCombiner` has the following config schema:
 
 ```python
 import ludwig.utils.schema_utils as schema
```

Comment:
maybe add the import line for `BaseCombinerConfig` (since I'm recommending that people define a new module instead of putting their new combiners in combiners.py).  They'll have to import it from combiners.

Reply:
added